benchmarks/config.h

```diff
@@ -42,6 +43,38 @@ struct AppConfig
     std::string benchmark_color; //  {auto|true|false}
     std::string benchmark_counters_tabular;
     std::string v; // <verbosity>
+
+    std::string get_input_path(const char* default_value = "generated/tiff_stripe_4096x4096_256.tif") const
```

Comment:
Should `default_value` be `std::string`? Similar lines below

Reply:
Thanks! Updated to `const std::string`